clients/drcachesim/tracer/instru_online.cpp

```diff
@@ -332,6 +332,15 @@ online_instru_t::insert_save_type_and_size(void *drcontext, instrlist_t *ilist,
         MINSERT(ilist, where,
                 XINST_CREATE_store(drcontext, OPND_CREATE_MEM32(base, disp),
                                    opnd_create_reg(scratch)));
+#elif defined(RISCV64)
```

Comment:
Not related to this PR: there is someone asking about building DR for RISC-V on the users list: https://groups.google.com/g/dynamorio-users/c/7LYfNgD8OEA

Hoping you or someone in your group could help answer it.

Reply:
Ack.